DataFormats/Detectors/TRD/include/DataFormatsTRD/CalVdriftExB.h

```diff
@@ -34,7 +34,7 @@ class CalVdriftExB
   void setVdrift(int iDet, float vd) { mVdrift[iDet] = vd; }
   void setExB(int iDet, float exb) { mExB[iDet] = exb; }
 
-  float getVdrift(int iDet) const { return mVdrift[iDet]; }
+  float getVdrift(int iDet) const { return (mVdrift[iDet] > 0.0) ? mVdrift[iDet] : constants::VDRIFTDEFAULT; }
```

Comment:
This I don't like. I think the CCDB object should always give you what is stored inside. And then it would be up to the consumer to check if the value is 0 or not and possibly use the default value then